src/libs/ajax/Metrics.ts

```diff
@@ -34,8 +34,8 @@ export const Metrics = (signal?: AbortSignal) => {
         appId: 'Saturn',
         hostname: window.location.hostname,
         appPath: Nav.getCurrentRoute().name,
-        appVersion: `https://github.com/DataBiosphere/terra-ui/commits/${getConfig().gitRevision}`,
-        appVersionPublishDate: new Date(parseInt(getConfig().buildTimestamp, 10)).toLocaleString(),
```

Comment:
This value is already an integer, not a string, so there's no need to `parseInt`.
https://app.terra.bio/build-info.json